staging_vespalib/src/vespa/vespalib/util/scheduledexecutor.h

```diff
@@ -17,7 +17,7 @@ class TimerTask;
  * interval. The timer can be reset to clear all tasks currently being
  * scheduled.
  */
-class Timer
+class ScheduledExecutor
```

Comment:
Consider updating class comments to reflect the new name

Reply:
Fixed